rotkehlchen/chain/ethereum/modules/makerdao/constants.py

```diff
@@ -10,6 +10,7 @@
 CPT_VAULT = 'makerdao vault'
 CPT_DSR = 'makerdao dsr'
 CPT_MIGRATION = 'makerdao migration'
+CPT_SDAI = 'sDAI protocol'
```

Comment:
this is the name of the protocol

```suggestion
CPT_SDAI = 'sDAI'
```